overlays/holo-nixpkgs/hpos-admin/hpos-admin.py

```diff
@@ -0,0 +1,87 @@
+from base64 import b64encode
+from flask import Flask, jsonify, request
+from gevent import subprocess, pywsgi, queue, spawn, lock
+from hashlib import sha512
+from tempfile import mkstemp
+import json
+import logging
+import os
+
+app = Flask(__name__)
+log = logging.getLogger(__name__)
+rebuild_queue = queue.PriorityQueue()
+lock = lock.Semaphore(1)
```

Comment:
`lock.Semaphore` default is `1`. Also, `lock` here is not a descriptive global variable and clashes with import (it should be `state_lock` instead), and this section was alphasorted.

```suggestion
global_lock = lock.Semaphore()
```